.env.example

```
@@ -6,6 +6,7 @@ PUBLIC_AUTH_BASE_URL=https://auth.openfoodfacts.net
 PUBLIC_AUTH_PKCE_ID=explorer
 PUBLIC_KEYCLOAK_REALM=openfoodfacts
 PUBLIC_PRICES_API_URL=https://prices.openfoodfacts.net
+PUBLIC_OPEN_PRICES_BASE_URL=https://prices.openfoodfacts.org
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PUBLIC_OPEN_PRICES_BASE_URL key should go before the PUBLIC_PRICES_API_URL key

(UnorderedKey)